### ISSUE TITLE
Release version 0.11 depends on modules with ~ version

### ISSUE THREAD

**Issue by likemute** [balderdashy/sails#3036]
@sgress454
Release version 0.11 depends on modules with ~ version and can lead to broken project when new instance is deployed on new server through npm install

Example: 
I have multiserver multiinstance project on production built on sails 0.11 . Today i was deploying it to a new server and got error with sockets/sessions (i have custom hooks to it, i wrote them referring to old 0.11 with sails-hook-sockets v."~0.11.21"). And now if i buld it with "npm install" i'm getting old 0.11 sails + new sails-hook-sockets "0.11.23" with changed code.

IMO a better practice to set specific dependency library versions on release.

**Comment by tjwebb** [comment]
Agree, I filed an issue on this awhile back. Let me find it...

**Comment by sailsbot** [comment]
Thanks for posting, @likemute.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
